libwallet/src/slate.rs

```diff
@@ -1088,8 +1106,8 @@ pub fn tx_from_slate_v4(slate: &SlateV4) -> Option<Transaction> {
 			}),
 		}
 	}
-	if slate.offset != BlindingFactor::zero() {
-		tx.offset = slate.offset.clone()
+	if slate.off != BlindingFactor::zero() {
+		tx.offset = slate.off.clone()
```

Comment:
What's the rationale for testing against zero? Is tx.offset already set to a nonzero value if slate.off == zero?

Reply:
Left over from an earlier iteration, thanks for spotting.